src/menus/FileMenus.cpp

```diff
@@ -66,7 +66,7 @@ void DoExport(AudacityProject &project, const FileExtension &format)
       // or we use the default documents folder - just as for exports.
       FilePath pathName = FileNames::FindDefaultPath(FileNames::Operation::MacrosOut);
 
-      if (!FileNames::WritableLocationCheck(pathName))
+      if (!FileNames::WritableLocationCheck(pathName, wxT("Cannot proceed to export,")))
```

Comment:
These and other strings need macro `XO` and changed punctuation.  When that is fixed, please list in the pull request comment what all the new strings in XO are.  Someone besides me might want to specify different words.